pkg/internal/local/klient/klient.go

```diff
@@ -0,0 +1,49 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package klient
+
+import (
+	"errors"
+	"os"
+	"path"
+
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// NewKubeClientFromConfig - creates a new Kubernetes CRUD client from a given kubeconfig
+func NewKubeClientFromConfig(configStr string) (client.Client, error) {
+	config, err := clientcmd.RESTConfigFromKubeConfig([]byte(configStr))
+	if err != nil {
+		return nil, err
+	}
+	return client.New(config, client.Options{Scheme: clientgoscheme.Scheme})
+}
+
+// NewKubeClientFromContext - creates a new Kubernetes CRUD client from the kubeconfig current-context
+func NewKubeClientFromContext() (client.Client, error) {
```

Comment:
We could refactor/export these functions to use a unified way to build k8s client. WDYT ?
https://github.com/cloudoperators/greenhouse/blob/ab63164e85e165d2fbedb9a5ba1a26ed0630a6bf/pkg/cmd/cluster_bootstrap.go#L707-L728

Reply:
Unused function I will remove it.